spec/models/tag_spec.rb

```diff
@@ -114,7 +113,7 @@ def make_some_tags(count: 3, tag_a_topic: false)
 
     it "returns all pm tags if user is a staff and pm tagging is enabled" do
       SiteSetting.allow_staff_to_tag_pms = true
-      expect(described_class.pm_tags(guardian: Guardian.new(Fabricate(:admin)))).to match_array(@private_tags.map(&:name))
+      expect(described_class.pm_tags(guardian: Guardian.new(Fabricate(:admin))).length).to eq(2)
```

Comment:
hmm testing for the length is not sufficient. We should also test for the objects we're expecting in the array.